.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,64 @@
+name: windows-ci
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  rust-version: 1.62.0
+  python-version: "3.9"
+  poetry-version: 1.1.13
+  sqlite-version: 3390100
+  winfsp-version: 1.8.20304
+  pytest-args: >-
+    --log-level=DEBUG
+    --durations=10
+    --side-effects-timeout=10
+
+jobs:
+  python-only:
+    runs-on: windows-2022
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # pin v3.0.2
+
+      - name: Install WinFSP
+        shell: bash
+        run: |
+          choco install -y --limit-output winfsp --version=${{ env.winfsp-version }}
+          mkdir winfsp-test
+          curl -L https://github.com/billziss-gh/winfsp/releases/download/v1.8/winfsp-tests-1.8.20304.zip -o winfsp-test/winfsp-tests.zip
+          unzip winfsp-test/winfsp-tests.zip -d winfsp-test
+          pwd
+          echo "$(pwd)\winfsp-test" >> "$GITHUB_PATH"
+
+      - name: Install poetry
+        shell: bash
+        run: |
+          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py \
+            | python - --version=${{ env.poetry-version }}
+          echo "$USERPROFILE\.poetry\bin" >> "$GITHUB_PATH"
+
+      - uses: actions/setup-python@d09bd5e6005b175076f227b13d9730d56e9dcfcb  # pin v4.0.0
+        with:
+          python-version: ${{ env.python-version }}
+          cache: poetry
+
+      - name: Install python deps
+        run: poetry install -E core -E backend
+
+      - name: Generate PyQT GUI data
+        run: poetry run python misc/generate_pyqt.py
+
+      - name: Test fast
+        run: poetry run pytest tests ${{ env.pytest-args }} -n auto
```

Comment:
Can you use `-x` also on fast tests ? this makes the build fail faster and avoids huge error log when something is definitely wrong (e.g. https://github.com/Scille/parsec-cloud/runs/7437053385?check_suite_focus=true)

Reply:
done in 4c64024